Library/Format.swift

```diff
@@ -141,38 +141,46 @@ public enum Format {
    */
   // swiftlint:disable valid_docs
   public static func duration(
-    secondsInUTC seconds: NSTimeInterval,
+    secondsInUTC seconds: TimeInterval,
                  abbreviate: Bool = false,
                  useToGo: Bool = false,
                  env: Environment = AppEnvironment.current) -> (time: String, unit: String) {
 
-    let components = env.calendar.components([.Day, .Hour, .Minute, .Second],
-                                             fromDate: env.dateType.init().date,
-                                             toDate: env.dateType.init(timeIntervalSince1970: seconds).date,
-                                             options: [])
+    let components = env.calendar.dateComponents([.day, .hour, .minute, .second],
+                                                 from: env.dateType.init().date,
+                                                 to: env.dateType.init(timeIntervalSince1970: seconds).date)
+
+    let (day, hour, minute, second) = (components.day ?? 0,
+                                       components.hour ?? 0,
+                                       components.minute ?? 0,
+                                       components.second ?? 0)
```

Comment:
swift 3 doesnt have optional comparisons (which is good!), so coalescing these separately so they can be used down below more easily...